app-dicts/myspell-sv/myspell-sv-2.14.ebuild

```diff
@@ -28,7 +28,7 @@ HOMEPAGE="
 	https://extensions.libreoffice.org/extension-center/swedish-thesaurus-based-on-synlex
 "
 SRC_URI="
-	https://extensions.libreoffice.org/extension-center/swedish-spelling-dictionary-den-stora-svenska-ordlistan/releases/${PV}/ooo_swedish_dict_${PV}.oxt
+	https://extensions.libreoffice.org/extension-center/swedish-spelling-dictionary-den-stora-svenska-ordlistan/releases/${PV}/ooo_swedish_dict_${PV}.oxt -> ${P}-dict.oxt
```

Comment:
Don't know if its related, or if it never worked, but
```
>>> Emerging (1 of 1) app-dicts/myspell-sv-2.14::gentoo
>>> Failed to emerge app-dicts/myspell-sv-2.14, Log file:
>>>  '/var/tmp/portage/vbslogs/build/app-dicts/myspell-sv-2.14:20210603-101534.log'
>>> Jobs: 0 of 1 complete, 1 failed                 Load avg: 0.75, 0.60, 0.29
 * Package:    app-dicts/myspell-sv-2.14
 * Repository: gentoo
 * USE:        abi_x86_64 amd64 elibc_glibc kernel_linux userland_GNU
 * FEATURES:   network-sandbox preserve-libs sandbox userpriv usersandbox

>>> Unpacking source...
>>> Unpacking /var/tmp/portage/app-dicts/myspell-sv-2.14/distdir/myspell-sv-2.14-dict.oxt to /var/tmp/portage/app-dicts/myspell-sv-2.14/work
[/var/tmp/portage/app-dicts/myspell-sv-2.14/distdir/myspell-sv-2.14-dict.oxt]
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of /var/tmp/portage/app-dicts/myspell-sv-2.14/distdir/myspell-sv-2.14-dict.oxt or
        /var/tmp/portage/app-dicts/myspell-sv-2.14/distdir/myspell-sv-2.14-dict.oxt.zip, and cannot find /var/tmp/portage/app-dicts/myspell-sv-2.14/distdir/myspell-sv-2.14-dict.oxt.ZIP, period.
 * ERROR: app-dicts/myspell-sv-2.14::gentoo failed (unpack phase):
 *   failed unpacking /var/tmp/portage/app-dicts/myspell-sv-2.14/distdir/myspell-sv-2.14-dict.oxt
 * 
 * Call stack:
 *               ebuild.sh, line 127:  Called src_unpack
 *             environment, line 270:  Called myspell-r2_src_unpack
 *             environment, line 236:  Called assert 'failed unpacking /var/tmp/portage/app-dicts/myspell-sv-2.14/distdir/myspell-sv-2.14-dict.oxt'
 *   isolated-functions.sh, line  18:  Called die
 * The specific snippet of code:
 *   		[[ $x -eq 0 ]] || die "$@"
 * 
 * If you need support, post the output of `emerge --info '=app-dicts/myspell-sv-2.14::gentoo'`,
 * the complete build log and the output of `emerge -pqv '=app-dicts/myspell-sv-2.14::gentoo'`.
 * The complete build log is located at '/var/tmp/portage/vbslogs/build/app-dicts/myspell-sv-2.14:20210603-101534.log'.
 * For convenience, a symlink to the build log is located at '/var/tmp/portage/app-dicts/myspell-sv-2.14/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/app-dicts/myspell-sv-2.14/temp/environment'.
 * Working directory: '/var/tmp/portage/app-dicts/myspell-sv-2.14/work'
 * S: '/var/tmp/portage/app-dicts/myspell-sv-2.14/work'
```

Reply:
Link is broken, tried now (I haven't before because I didn't have checked old version of myspell)